tests/test_cli.py

```diff
@@ -13,6 +13,14 @@
     from pathlib import Path
 
 
+def test_cli_version(capsys: pytest.CaptureFixture[str]) -> None:
+    with pytest.raises(SystemExit) as context:
+        cli_args(["--version"])
+    assert context.value.code == 0
+    out, err = capsys.readouterr()
+    assert out.startswith(("pytest", "__main__.py"))
```

Comment:
Shouldn't this be pyproject-fmt? Perhaps we should set prog above.

Reply:
Also there's no assert that it prints out the correct version.

Reply:
a) The patch has been amended to define the `prog` parameter to `ArgumentParser`. Is it the right choice?
b) The test case has been adjusted to also verify the correct version number is actually used.